crates/task/src/dependency_task.rs

```diff
@@ -82,10 +82,7 @@ mod test {
     }
 
     fn eq_dep(rx: Receiver<usize>, val: usize) -> EventDependency<usize> {
-        EventDependency {
-            event_rx: rx,
-            match_fn: Box::new(move |v| *v == val),
-        }
+        EventDependency::new(rx, Box::new(move |v| *v == val))
```

Comment:
Had to swap this to account for the new field being private, so the `new` constructor was needed.